docs/quickstarts/csharp-quickstart.md

```diff
@@ -0,0 +1,254 @@
+* Mac OS X: Xamarin Studio 5.9+
+
+### How to use Grpc
+
+The example solution in this walkthrough already adds the necessary
+dependencies on Grpc, but this explains how to use Grpc in your projects.
+
+**Windows**
+
+- Open Visual Studio and start a new project/solution. 
+- Add NuGet package `Grpc` as a dependency (Project options -> Manage NuGet Packages). (Make sure to check the box to "Include prerelease" to get the pre-release version).
+  That will also pull all the transitive dependencies (including the gRPC native library that
+  gRPC C# is using internally).
+  
+**Linux (Debian)**
+
+- Open MonoDevelop and start a new project/solution.
+
+- Add NuGet package `Grpc` as a dependency (Project -> Add NuGet packages). (Make sure to check the box to "Show pre-release packages" to ge the pre-release version).
+  That will also pull all the transitive dependencies (including the gRPC native library that
+  gRPC C# is using internally).
+
+**Mac OS X**
+
+- Open Xamarin Studio and start a new project/solution.
+
+- Add NuGet package `Grpc` as a dependency (Project -> Add NuGet packages). (Make sure to check the box to "Show pre-release packages" to ge the pre-release version). 
+That will also pull all the transitive dependencies (including the gRPC native library thatgRPC C# is using internally).
+  
+Example projects depend on the Grpc, Grpc.Tools and Google.Protobuf NuGet packages, which have been already added to the project for you.
+
+## Download the example
+
+You'll need a local copy of the example code to work through this quickstart. 
+Download the example code from our Github repository (the following command clones the entire repository, but you just need the examples for this quickstart and other tutorials):
+
+```sh
+$ # Clone the repository to get the example code:
+$ git clone -b $(curl -L http://grpc.io/release) https://github.com/grpc/grpc 
+```
+
+All of the files in this quickstart are in the directory `examples/csharp/helloworld`.
+
+## Build the example
+
+* Open solution `Greeter.sln` with Visual Studio, Monodevelop (on Linux) or Xamarin Studio (on Mac OS X)
+* Build the solution (this will automatically download NuGet dependencies)
+  
+## Run a gRPC application
+
+From the `examples/csharp/helloworld` directory:
+
+1. Run the server
+
+```
+> cd GreeterServer/bin/Debug
+> GreeterServer.exe
+```
+
+2. In another terminal, run the client
+
+```
+> cd GreeterClient/bin/Debug
+> GreeterClient.exe
+```
+
+Note to run the above executables with "mono" if building on Xamarin Studio for Mac OS X.
+
+Congratulations! You've just run a client-server application with gRPC.
+
+## Update a gRPC service
+
+Now let's look at how to update the application with an extra method on the
+server for the client to call. Our gRPC service is defined using protocol
+buffers; you can find out lots more about how to define a service in a `.proto`
+file in [What is gRPC?]() and [gRPC Basics: C#][]. For now all you need
```

Comment:
We can drop the reference to what is gRPC here. ie: you can find out lot more about how to define a service in a `.proto` file in [gRPC Basics: C#][].